lib/config.js

```diff
@@ -69,8 +69,8 @@ module.exports = {
         team: 10 * 60, // 10 mins
       },
       permissions: {
-        harvest: ['harvesters'],
-        curate: ['curators'],
+        harvest: [config.get('AUTH_HARVEST_TEAM')].filter(e => e),
```

Comment:
What's the filter accomplish here?

Reply:
removes nulls. if the env var is not set then `get` returns null and we end up with `[ null ]`. Elsewhere in the code we use length === 0 to indicate that no permissions checking is needed.